x-pack/plugins/snapshot_restore/public/app/sections/home/home.tsx

```diff
@@ -0,0 +1,152 @@
+
+import { RepositoryList } from './repository_list';
+import { SnapshotList } from './snapshot_list';
+import { documentationLinksService } from '../../services/documentation';
+
+interface MatchParams {
+  section: Section;
+}
+
+export const SnapshotRestoreHome: React.FunctionComponent<RouteComponentProps<MatchParams>> = ({
+  match: {
+    params: { section },
+  },
+  history,
+}) => {
+  const {
+    core: {
+      i18n: { FormattedMessage },
+    },
+  } = useAppDependencies();
+
+  const tabs = [
+    {
+      id: 'snapshots' as Section,
+      name: (
+        <FormattedMessage
+          id="xpack.snapshotRestore.home.snapshotsTabTitle"
+          defaultMessage="Snapshots"
+        />
+      ),
+      testSubj: 'srSnapshotsTab',
+    },
+    {
+      id: 'repositories' as Section,
+      name: (
+        <FormattedMessage
+          id="xpack.snapshotRestore.home.repositoriesTabTitle"
+          defaultMessage="Repositories"
+        />
+      ),
+      testSubj: 'srRepositoriesTab',
+    },
+  ];
+
+  const onSectionChange = (newSection: Section) => {
+    history.push(`${BASE_PATH}/${newSection}`);
+  };
+
+  // Set breadcrumb
+  useEffect(() => {
+    breadcrumbService.setBreadcrumbs('home');
+  }, []);
+
+  return (
+    <EuiPageBody>
+      <EuiPageContent>
+        <EuiTitle size="l">
+          <EuiFlexGroup alignItems="center">
+            <EuiFlexItem grow={true}>
+              <h1>
+                <FormattedMessage
+                  id="xpack.snapshotRestore.home.snapshotRestoreTitle"
+                  defaultMessage="Snapshot Repositories"
+                />
+              </h1>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiButtonEmpty
+                href={documentationLinksService.getRepositoryTypeDocUrl()}
+                target="_blank"
+                iconType="help"
+              >
+                <FormattedMessage
+                  id="xpack.snapshotRestore.home.snapshotRestoreDocsLinkText"
+                  defaultMessage="Snapshot docs"
+                />
+              </EuiButtonEmpty>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiTitle>
+        <EuiSpacer size="s" />
+        <EuiTitle size="s">
+          <EuiText color="subdued">
+            <FormattedMessage
+              id="xpack.snapshotRestore.home.snapshotRestoreDescription"
+              defaultMessage="Use repositories to store backups of your Elasticsearch indices and clusters."
+            />
+          </EuiText>
+        </EuiTitle>
+
+        <EuiSpacer size="m" />
+
+        <EuiTabs>
+          {tabs.map(tab => (
+            <EuiTab
+              onClick={() => onSectionChange(tab.id)}
+              isSelected={tab.id === section}
+              key={tab.id}
+              data-test-subject={tab.testSubj}
+            >
+              {tab.name}
+            </EuiTab>
+          ))}
+        </EuiTabs>
+
+        <EuiSpacer size="m" />
+
+        <Switch>
```

Comment:
Should we add a `Redirect` case here if the section name isn't `snapshots` or `repositories`?

Reply:
good idea!